bindings/python/CMakeLists.txt

```diff
@@ -24,7 +24,8 @@ file(
   tasks/*.cpp
   trajectories/*.cpp
   math/*.cpp
-  utils/*.cpp)
+  utils/*.cpp
+  measuredForces/*.cpp)
```

Comment:
For consistency I think also the bindings files should go in the contacts folder now